README.md

```diff
@@ -83,52 +85,46 @@ Where `source` is the file location.
 Having the following workflow instance:
 
 ```typescript
-    const workflow = new WorkflowBuilder()
-        .withId("helloworld")
-        .withVersion("1.0")
-        .withName("Hello World Workflow")
-        .withDescription("Inject Hello World")
-        .withStart("Hello State")
-        .withStates([new InjectStateBuilder()
-            .withName("Hello State")
-            .withData({
-                "result": "Hello World!"
-            })
-            .withEnd(true).build()])
-        .build();
+const workflow = workflowBuilder()
+  .id("helloworld")
+  .version("1.0")
+  .name("Hello World Workflow")
+  .description("Inject Hello World")
+  .start("Hello State")
+  .states([injectstateBuilder()
+    .type("inject")
+    .name("Hello State")
+    .data({
+      "result": "Hello World!"
+    })
+    .end(true)
+    .build()
```

Comment:
And maybe another stupid question, sorry- are these "build()" methods absolutely required?
For larger workflows thats alot of "build()" entries.

Reply:
Yes, `build` is a magic accessor that will trigger the proxy validation function and return the underlying object.